include/utils.h

```diff
@@ -172,7 +172,7 @@ inline void convert_labels_string_to_int(const std::string &inFileName, const st
             token.erase(std::remove(token.begin(), token.end(), '\r'), token.end());
             if (string_int_map.find(token) == string_int_map.end())
             {
-                uint32_t nextId = (uint32_t)string_int_map.size() + 1;
```

Comment:
I see. Hope we don't make any assumptions elsewhere in the code which hardcodes 0 for universal label. Did you check this? Currently, if UNV is found, we have 0, 2, 3, ... and otherwise we have 1, 2, 3, .. for integer range.

Reply:
this function always assign "0" to universal label. so when UNV is found, it is the same as before. otherwise, we should not use "0" as special id

Reply:
yes, makes sense. i just want to make sure we dnt assume that 0 means universal label in other places of code. will confirm once and approve.